src/EdgeAnalysis.jl

```diff
@@ -305,18 +309,28 @@ function processfluxes(sim::SystemSimulation,
             if @inbounds any(d.rxnarray[:,i].>length(corespeciesconcentrations))
                 continue
             end
+            net_consumption = max(frts[i+index]-rrts[i+index], 0.0)
             for j = 1:4
                 if @inbounds d.rxnarray[j,i] != 0
                     @inbounds corespeciesconsumptionrates[d.rxnarray[j,i]] += frts[i+index]
                     @inbounds corespeciesproductionrates[d.rxnarray[j,i]] += rrts[i+index]
+                    if d.species[d.rxnarray[j,i]].radicalelectrons == 1
+                        coreradicalnetconsumptionrates[d.rxnarray[j,i]] += net_consumption
```

Comment:
rname and do consumption vector for all species?

Reply:
Done